packages/swapper/src/thorchain-utils/memo/assertAndProcessMemo.ts

```diff
@@ -180,14 +180,14 @@ export const assertAndProcessMemo = (memo: string): string => {
       if (pool.includes('.')) {
         if (maybeAsset) assertMemoHasAsset(maybeAsset, memo)
         assertIsValidBasisPoints(basisPoints, memo)
-        return `${_action}:${pool}:${basisPoints}:${maybeAsset ?? ''}:${THORCHAIN_AFFILIATE_NAME}:0`
+        return `${_action}:${pool}:${basisPoints}:${maybeAsset ?? ''}`
       }
 
       // Withdraw Savers - WITHDRAW:POOL:BASISPOINTS
       if (pool.includes('/')) {
         if (maybeAsset) throw new Error('asset is not supported for savers withdraw')
         assertIsValidBasisPoints(basisPoints, memo)
-        return `${_action}:${pool}:${basisPoints}::${THORCHAIN_AFFILIATE_NAME}:0`
+        return `${_action}:${pool}:${basisPoints}`
```

Comment:
Savers withdraw were never borked as a result of affiliate name/0 bps, we need to keep this for tracking purposes

Reply:
Reverted in [cef1d99](https://github.com/shapeshift/web/pull/7351/commits/cef1d9921a4902ef2281ae8dc06a883d474dbfd0)